bulletin_board/server/config/environments/production.rb

```diff
@@ -38,7 +38,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon
```

Comment:
This is against Decidim Social Contract. You can't force implementers to use a privative external service. Please change it and do it an optional env var

Reply:
Reported at #261